Log out all devices (#226)
NabuCasa/support: content/src/cloud/nabu-casa-account/log-out-all-devices.md

---
zendesk:
  article_id: 26170426881693
  name: Logging out all devices from Home Assistant Cloud
  position: 24
  labels: cloud
---

> You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, browsers, and the Home Assistant Companion App on any of your phones or tablets.

Reviewer: You are not logged into your nabucasa account in the companion apps?

Author: Thank you!

> You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, browsers, and the Home Assistant Companion App on any of your phones or tablets.

Author: ```suggestion
You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, and browsers.
```

---
zendesk:
  article_id: 26170426881693
  name: Logging out all devices from Home Assistant Cloud
  position: 24
  labels: cloud
---

You can log out all devices that are currently logged in to Home Assistant Cloud. This includes Home Assistant, Google Home, Alexa, and browsers.

>**Notice** If you log out of your device, your remote connection stops working
>

> > You need physical access to home assistant to renable remote connection.

Author: ```suggestion
> You need physical access to Home Assistant to re-establish a remote connection.
```

> > You need physical access to Home Assistant to re-establish a remote connection.

Reviewer: ```suggestion
> You need physical access to Home Assistant to re-establish the remote connection.
```
